app/models/reports/bulk_recommend_report.rb

```diff
@@ -2,7 +2,8 @@
 
 module Reports
   class BulkRecommendReport < TemplateClassCsv
-    DO_NOT_EDIT = "Do not edit"
+    DO_NOT_EDIT   = "Do not edit"
+    DATE_GUIDANCE = "For example, 20/7/2022/n/nDelete row if the trainee has not met the standards"
```

Comment:
When I open this in numbers, the double `/n/n` makes the "Delete row" bit fall out the bottom of the cell 😂 

<img width="759" alt="Screenshot 2023-04-03 at 18 12 38" src="https://user-images.githubusercontent.com/18436946/229580381-d38f4416-7578-460e-ae80-3c110ae1247c.png">

Reply:
I've used `\r\n` instead and this seems to have fixed it